docs/docsite/rst/dev_guide/platforms/vmware_guidelines.rst

```diff
@@ -1,262 +1 @@
-
-Use FQCN for examples within module documentation. For instance, you should use ``community.vmware.vmware_guest`` instead of just
-``vmware_guest``.
-
-This way, the examples do not depend on the ``collections`` directive of the
-playbook.
-
-Functional tests
-----------------
-
-Writing new tests
-^^^^^^^^^^^^^^^^^
-
-If you are writing a new collection of integration tests, there are a few VMware-specific things to note beyond
-the standard Ansible :ref:`integration testing<testing_integration>` process.
-
-The test-suite uses a set of common, pre-defined vars located `in prepare_vmware_tests <https://github.com/ansible-collections/community.vmware/tree/main/tests/integration/targets/test/integration/targets/prepare_vmware_tests/>`_ role.
-The resources defined there are automatically created by importing that role at the start of your test:
-
-.. code-block:: yaml
-
-  - import_role:
-      name: prepare_vmware_tests
-    vars:
-      setup_datacenter: true
-
-This will give you a ready to use cluster, datacenter, datastores, folder, switch, dvswitch, ESXi hosts, and VMs.
-
-No need to create too much resources
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Most of the time, it's not necessary to use ``with_items`` to create multiple resources. By avoiding it,
-you speed up the test execution and you simplify the clean up afterwards.
-
-VM names should be predictable
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-If you need to create a new VM during your test, you can use ``test_vm1``, ``test_vm2`` or ``test_vm3``. This
-way it will be automatically clean up for you.
-
-Avoid the common boiler plate code in your test playbook
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-From Ansible 2.10, the test suite uses `modules_defaults`. This module
-allow us to preinitialize the following default keys of the VMware modules:
-
-- hostname
-- username
-- password
-- validate_certs
-
-For example, the following block:
-
-.. code-block:: yaml
-
-    - name: Add a VMware vSwitch
-      community.vmware.vmware_vswitch:
-        hostname: '{{ vcenter_hostname }}'
-        username: '{{ vcenter_username }}'
-        password: '{{ vcenter_password }}'
-        validate_certs: 'no'
-        esxi_hostname: 'esxi1'
-        switch_name: "boby"
-        state: present
-
-should be simplified to just:
-
-.. code-block:: yaml
-
-    - name: Add a VMware vSwitch
-      community.vmware.vmware_vswitch:
-        esxi_hostname: 'esxi1'
-        switch_name: "boby"
-        state: present
-
-
-Typographic convention
-======================
-
-Nomenclature
-------------
-
-We try to enforce the following rules in our documentation:
-
-- VMware, not VMWare or vmware
-- ESXi, not esxi or ESXI
-- vCenter, not vcenter or VCenter
-
-We also refer to vcsim's Go implementation with ``govcsim``. This to avoid any confusion with the outdated implementation.
+This guide has moved to :ref:ansible_collections.community.vmware.docsite.vmware_ansible_devguide
```

Comment:
```suggestion
.. _VMware_module_development:

******************************************************
Guidelines for VMware module development
******************************************************

This guide has moved to :ref:`ansible_collections.community.vmware.docsite.vmware_ansible_devguide`.
```

Reply:
This suggested edit should fix the CI error... basically need to keep the title and old anchor in the file.

Reply:
LOL I just wanted to ask you if it was OK to remove the anchor! It just occurred to me that this might have been a mistake :-)